packages/patternlab/styleguide/source/assets/js/modules/imageFill.js

```diff
@@ -8,7 +8,7 @@ export default (function (window, document, $, undefined) {
 
   function mediaWidth() {
     // Define wrapper width for use.
-    var wrapperWidth = $(".main-content").width();
+    var wrapperWidth = $("#main-content > .main-content").width();
```

Comment:
```suggestion
    var wrapperWidth = $("#main-content > .main-content--two").width();
```

I feel like it's more accurate to target `.main-content--two`, as this class sets the container width. e.g. `main-content main-content--full` is not the right container width

Reply:
or `$("#main-content > .main-content.main-content--two")`